app/views/questions/show.html.erb

```diff
@@ -12,7 +12,7 @@
       <h2 class="title-no-margin"><%= @question.text %></h2>
       <br>
       <%= form_for @new_answer, :url => question_answers_path(@question.id), remote: true do |f| %>
-        <%= f.text_area :text, autofocus: true, placeholder: 'Type your answer here', class: 'form-control answer-text', id: 'single-answer-text' %>
+        <%= f.text_area :text, autofocus: true, placeholder: 'Type your answer here', class: 'form-control answer-text center', id: 'single-answer-text' %>
```

Comment:
This `center` class is making the text in the textarea centered, not centering the textarea in its context (see screenshot).
<img width="704" alt="screen shot 2015-09-25 at 9 09 15 pm" src="https://cloud.githubusercontent.com/assets/8975317/10115030/f27ee670-63c9-11e5-8135-cc877226bac5.png">

If I remove `center` then it looks like the modal dialog, so I think that's all we need to do here.

(These don't look perfect, of course -- the textarea isn't stretched or centered properly in the containing element, so if you can fix that great, but you don't have to as part of this PR; the UI may change a bit anyway once we complete our UX redesign.)